services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/application/ApplicationControllerITest.kt

```diff
@@ -12,7 +12,12 @@ import fi.hel.haitaton.hanke.HankeService
 import fi.hel.haitaton.hanke.IntegrationTestConfiguration
 import fi.hel.haitaton.hanke.OBJECT_MAPPER
 import fi.hel.haitaton.hanke.andReturnBody
-import fi.hel.haitaton.hanke.factory.AlluDataFactory
+import fi.hel.haitaton.hanke.factory.AlluDataFactory.Companion.cableReportWithoutHanke
+import fi.hel.haitaton.hanke.factory.AlluDataFactory.Companion.createApplication
+import fi.hel.haitaton.hanke.factory.AlluDataFactory.Companion.createApplications
+import fi.hel.haitaton.hanke.factory.AlluDataFactory.Companion.createCableReportApplicationData
+import fi.hel.haitaton.hanke.factory.AlluDataFactory.Companion.createCompanyCustomer
+import fi.hel.haitaton.hanke.factory.AlluDataFactory.Companion.createContact
```

Comment:
I think static imports should be used sparingly. Just importing every static method you can doesn't make the code easier to read. It forces the reader to think things like "Where is this method defined?" when it could be clear from the context. This increases the cognitive load when reading code since the reader needs to keep which methods are imported in their head.

Like if you see `createApplications()` you don't know if this is a factory method or a local helper. The implications are different in each case, so this matters. So every time you encounter this call, the reader needs to remember "Oh yeah, that's a factory method."

If this is replaced with `AlluDataFactory.createApplications()` it's clear that this is a factory method and implies it produces a generic list of applications.

This doesn't mean I think static imports shouldn't be used, but there needs to be some concrete benefit to using them that outweighs the increased cognitive load.

Reply:
Brought back AlluDataFactory.createApplications.